services/app/src/components/team/Team.jsx

```diff
@@ -1,13 +1,21 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
+import { ChessBoardContext } from '../../contexts/ChessBoardContext';
 import TeamMember from './TeamMember';
 import GameStatus from '../gameStatus/GameStatus';
 import GameInfo from '../gameStatus/GameInfo';
 import playerPropType from '../../prop-types/player';
 import Return from './teamComponents/Return.svg';
 import './team.css';
 
-function Team({ players, skipMove, children, removePlayer, playerCount }) {
+function Team({ players, skipMove, children, removePlayer, playerCount, canPlay, clearBoard }) {
+  const { turns } = useContext(ChessBoardContext);
+
+  const restartBtnChecker = () => {
```

Comment:
There are a couple of issues with this function
* `restartBtnChecker` is not a suitable name, as it not descriptive enough. 
* Function is not what you should be using in this case, I'd strongly suggest to use `useMemo` hook in this place which would have exactly the same body, as this function, but it would not be ran at every single render, only when dependencies are changed.